docgen/src/guide/Migration_guide_package_import.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Migration Guide - Package import
+mainTitle: Guides
+layout: main.pug
+category: guide
+navWeight: 10
+---
+
+With the release of the version `5.2.0` we completely revamped the architecture of the package. We tried to keep the package backward compatibility as possible. The only case **where changes are required** is when you are using the private imports. Like the following example:
```

Comment:
"to keep the package backward compatibility as possible" → "to keep the package backward compatible"?